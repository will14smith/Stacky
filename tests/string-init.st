main () -> () { 5 !@ "abcde" 0 !!~ print }