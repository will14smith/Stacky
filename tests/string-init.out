=== Output ===
abcde

=== Exit ===
0
